micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/pages/employee/ProductDetails/ProductDetailsComponentUpdated.js

```
@@ -0,0 +1,347 @@
+        ? "https://example.com"
+        : "https://www.wikipedia.org";
+
+    return (
+        <div className="walkthrough-container" style={{ width: '100%', backgroundColor: '#ffffff', padding: '3rem 6rem' }}>
+            <div className="wt-c1">
+                <h2 className="wt-title">{t("SB_WALK_THROUHG_HEADER")}</h2>
+                <p className="wt-subtitle">{t("SB_WALK_THROUHG_DESCRIPTION")}</p>
+            </div>
+            <div className="wt-tabs-center wt-tabs-and-iframe">
+                <div className="wt-tab-wrapper">
+                    <div className={`wt-tab ${activeTab === "citizen" ? "active" : ""}`} onClick={() => setActiveTab("citizen")}>{t("SB_WALK_THROUHG_CITIZEN")}</div>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)